cmd/bblfshd/main.go

```diff
@@ -75,6 +79,7 @@ func init() {
 	log.fields = cmd.String("log-fields", "", "extra fields to add to every log line in json format.")
 	pprof.enabled = cmd.Bool("profiler", false, "run profiler http endpoint (pprof).")
 	pprof.address = cmd.String("profiler-address", ":6060", "profiler address to listen on.")
```

Comment:
It would be great to merge those two, but if exposing metrics might be okay, exposing profile might be not. So they probably will need different ports anyway.

Reply:
Well, if we don't want to expose profiles, we would just not add the handler for them, right?

Reply:
I thought it might be useful to host metrics on all interfaces (`:2112`) while exposing profiles only to localhost (`localhost:6060`). If it's not the case, I can try merging them.

Also, the problem is that the [`http/pprof`](https://golang.org/pkg/net/http/pprof/) package does not expose a single handler. I will have to emulate the same handler as used in the package internals.

Reply:
Ah, I see. In that case it might be more trouble than it's worth.

Reply:
I hadn't looked at the pprof API in a while, it looks like we could maybe use https://godoc.org/net/http/pprof#Index to avoid having to export multiple routes manually.

Reply:
You are right, but [`init`](https://golang.org/src/net/http/pprof/pprof.go#L72) function in the package does something different. I'm sure there will be more profiles added in the future, and we will have to sync that piece of code on our own. It's pretty small, but still.